src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/AnalyzeCovariates.java

```diff
@@ -167,43 +129,74 @@
     static final String PDF_ARG_SHORT_NAME = "plots";
     static final String BEFORE_ARG_SHORT_NAME = "before";
     static final String AFTER_ARG_SHORT_NAME = "after";
+    public static final String IGNORE_LMT_SHORT_NAME = "ignoreLMT";
```

Comment:
I would probably just have no short name for this option if the short name is going to be something weird cased and long.

Reply:
done